src/project-css/app/Helpers/CollectionHelper.php

```diff
@@ -0,0 +1,193 @@
+<?php
+/**
+ * @author Ajay Krishna Teja Kavur
+ * @author Tracy A McCormick <tam0013@mail.wvu.edu>
+ */
+
+namespace App\Helpers;
+
+use Illuminate\Support\Facades\Storage;
+use App\Models\Collection;
+
+class CollectionHelper {
+    /**
+     * Collection Helper
+     *
+     * These are various functions that help with creating and modifying 
+     * collections.
+     *
+     */
+
+     public function create($data) {
+        // Create the collection name
+        $thisClctn = new Collection;
+        $thisClctn->clctnName = $data['name'];
+        $thisClctn->isCms = $data['isCms'];
+
+        // set cmsId if it exists
+        if (array_key_exists('cmsId', $data)) {
+          $thisClctn->cmsId = $data['cmsId'];
+        }
+
+        $thisClctn->save();
+
+        // create folder in storage that will contain any additional files associated to the collection
+        if (Storage::exists($thisClctn->clctnName) == FALSE) {
+          Storage::makeDirectory($thisClctn->clctnName, 0775);
+        }
+
+        return $thisClctn;
+     }
+
+     public function update($data) {
+      // find the collection
+      $thisClctn = Collection::findorFail($data['id']);
+
+      if ($thisClctn->clctnName != $data['name']) {
+        // Rename Storage Folder
+        if ((Storage::exists($data['name']) == FALSE) && (Storage::exists($thisClctn->clctnName))) {
+          Storage::move($thisClctn->clctnName, $data['name']);
+        }  
+
+        // Set new Collection Name
+        $thisClctn->clctnName = $data['name'];
+      }
+
+      if ($thisClctn->isCms != $data['isCms']) {
+        // Set CMS State
+        $thisClctn->isCms = $data['isCms'];
+      }
+
+      // Save Updated items
+      $thisClctn->save();            
+     }
+
+     public function disable($name) {
+        // verify collection exists
+        if ($this->isCollection($name)) {
+          // find the collection
+          $thisClctn = Collection::where('clctnName', $name)->first();
+
+          // reset collection to disabled
+          $this->updateCollectionFlag($thisClctn->id, false);
+
+          return true;
+        }
+
+        return false;
+    }
+
+    public function enable($name) {
+      // verify collection exists
+      if ($this->isCollection($name)) {
+        // find the collection
+        $thisClctn = Collection::where('clctnName', $name)->first();
+
+        // reset collection to enabled
+        $this->updateCollectionFlag($thisClctn->id, true);
+
+        return true;
+      }
+
+      return false;
+     }
+
+     public function isCollection($name) {
+        if (Collection::where('clctnName', '=', $name)->count() == 1) {
+          return true;
+        }
+        return false;
+     }
+
+     public function setCMS($name, $option = false) {
```

Comment:
Needs Documentation: 
Be clear on what the parameters of the function are as well as what the return values are supposed to be.